docs/documentation/maya/simple_setup.md

```diff
@@ -1,221 +1,297 @@
 # A Simple Setup
 
-This page will be focused on displaying the process of creating and setting up each deformer in maya showing the process step by step.
+This page is dedicated to explain, step by step, a simple process of creating and setting every Adonis deformer in Maya. The scenarios presented here will are intended to provide the minimum required configurations to obtain plausible results.
 
-## AdnSkin simple setup
+## AdnSkin Simple Setup
 
-For AdnSkin we will start with a scene with the following elements:
+To create a basic scenario using the AdnSkin deformer start with a scene with the following elements:
 
   - A target mesh with animation.
   - A simulated mesh (without animation).
 
-The AdnSkin deformer will be applied to the simulated mesh.
+The AdnSkin deformer will get applied to the simulated mesh.
+
+<figure>
+  <img src="images/setup_skin_0.png"> 
+  <figcaption>Figure 1: Basic setup for skin simulations.</figcaption>
+</figure>
 
 ### 1. Creating the deformer
 
-To create the deformer we will select the target mesh and then the skin mesh (simulated mesh). Then we will press the ![adnMuscle](images/adn_skin.png) shelf button or go to *AdonisFX menu > Skin*.
+To create the AdnSkin deformer select the target mesh and then the skin mesh (simulated mesh). Then press the ![AdnMuscle](images/adn_skin.png) shelf button or go to AdonisFX Menu > *Skin*.
 
-If we want to create the AdnSkin deformer with some initial customization we can double click the shelf button or press the option box in the menu item. This will display a popup window that will allow us to do some initial customization, as well as creating the deformer with a custom name. Once everything is in order press the "Create" button and the deformer will get created.
+To create the AdnSkin deformer with some initial customization, double click the shelf button or press the option box in the menu item. This will display a pop-up window that will allow to do some initial customization, as well as creating the deformer with a custom name. Once all data has been provided press the "Create" button and the deformer will get created.
```

Comment:
To keep consistency:
```suggestion
To create the AdnSkin deformer with some initial customization, double-click the shelf button or press the option box in the menu item. This will display a pop-up window that will allow to do some initial customization, as well as creating the deformer with a custom name. Once all data has been provided press the *Create* button and the deformer will get created.
```

Reply:
Done.